hphp/runtime/vm/jit/vasm-arm.cpp

```diff
@@ -602,6 +603,10 @@ void Vgen::emit(const andli& i) {
   a->And(W(i.d), W(i.s1), MSKTOP(i.s0.l()), SetFlags);
 }
 
+void Vgen::emit(const testbi& i) {
+  a->Tst(W(i.s1), MSKTOP(0xff));
```

Comment:
I'm not really sure why you guys need this MSKTOP thing—I assume it's because you need 32-bit negative values to zero-extend instead of sign-extend when widened to `int64_t`.  Is that correct, @dave-estes?